File: javascript/Day_1/Question_06.js
const a = 5;
const b = 10;

const swapVariable = (a, b) => {
  temp = a;
  a = b;
  b = temp;
  return `swaping variable a is: ${a} and b is: ${b}`;
};

const result = swapVariable(a, b);

console.log(result);

File: dsa/Day_1/Question_06.cpp
#include <iostream>
using namespace std;

void swapVariable(int a, int b)
{
    int temp;
    temp = a;
    a = b;
    b = temp;

    cout << "new value of a and b is: " << a << "," << b << endl;
}

int main(){
    int a;
    int b;

    cout << "a : ";
    cin >> a;

    cout << "b : ";
    cin >> b;

    cout << "old value of a and b is: " << a << "," << b << endl;

    swapVariable(a, b);
}